perfkitbenchmarker/benchmarks/hadoop_terasort_benchmark.py

```diff
@@ -36,21 +36,23 @@
 FLAGS = flags.FLAGS
 
 BENCHMARK_INFO = {'name': 'hadoop_terasort',
-                  'description': 'Runs Terasort',
+                  'description': 'Runs Terasort. Control the number of VMs '
+                  'with --num_vms.',
```

Comment:
If this is shown as a usage summary, would be nice to show the default in the description, for example:
  `with --num_vms, default is %d' % DEFAULT_VM_COUNT`